FranticFour/Assets/01_Scripts/AssignPlayers.cs

```diff
@@ -1,8 +1,10 @@
 ﻿using UnityEngine;
 
+//Assigns the player from playerSelect
 public class AssignPlayers : MonoBehaviour
 {
-    [SerializeField] private AssignedController[] players = new AssignedController[4];
+    [SerializeField]
```

Comment:
Move [SerializeField] to the same line as the private AssignedController variable? 👍

Reply:
Fixing in next push :*